pom.xml

```diff
@@ -450,12 +450,14 @@
     </build>
 
     <properties>
-        <identity.framework.version>7.0.73</identity.framework.version>
+        <identity.framework.version>7.5.64</identity.framework.version>
+        <!-- TODO: Bump org.wso2.carbon.identity.testutil to same identity.framework.version-->
+        <identity.framework.testutil.version>7.0.73</identity.framework.testutil.version>
```

Comment:
Why we can't use the framework version itself

Reply:
Previous tests had failed due to the following change,

- https://github.com/wso2/carbon-identity-framework/commit/07a3b279e6505a5aea83f4c35fe81f2fea703891#diff-b89b10f23f13f6b87f1d0064acd26bd3906c5b8d0146ab160dec3c687ffca760L178

We may need to make significant changes to existing tests/dependencies in the repo to incorporate the changes properly.
IMO this should be done as a separate task.

Reply:
That dependency issue is resolved by this PR, https://github.com/wso2-extensions/identity-inbound-auth-sts/pull/165. So you should be able to use a single framework dependency.

Reply:
Fixed